spec/javascripts/health-care/components/ContinueButton.spec.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import SkinDeep from 'skin-deep';
+
+import ContinueButton from '../../../../_health-care/_js/components/ContinueButton';
+
+describe('<ContinueButton>', () => {
+  describe('propTypes', () => {
+    let consoleStub;
+    beforeEach(() => {
+      consoleStub = sinon.stub(console, 'error');
+    });
+
+    afterEach(() => {
+      consoleStub.restore();
+    });
+
+    // TODO: figure out why this is failing (same issue as in SSN spec)
+    xit('onButtonClick is required', () => {
+      SkinDeep.shallowRender(<ContinueButton/>);
+      sinon.assert.calledWithMatch(consoleStub, /Required prop `onButtonClick` was not specified in `ContinueButton`/);
+    });
+
+    it('onButtonClick must be a function', () => {
+      SkinDeep.shallowRender(
+        <ContinueButton onButtonClick/>);
+      sinon.assert.calledWithMatch(consoleStub, /Invalid prop `onButtonClick` of type `boolean` supplied to `ContinueButton`, expected `function`/);
+    });
+  });
+
```

Comment:
We should add a test for the onButtonClick() propagation. Let's do it in a follow-up PR.